test_runner/fixtures/pageserver/http.py

```diff
@@ -272,6 +272,7 @@ def timeline_create(
         new_timeline_id: Optional[TimelineId] = None,
         ancestor_timeline_id: Optional[TimelineId] = None,
         ancestor_start_lsn: Optional[Lsn] = None,
+        **kwargs,
```

Comment:
NIT: As I see is it used to pass only timeout, lets make it explicit with: `timeout: Optional[float] = None` or cal it request_timeout. Not insisting if you want to keep it as is, feel free to resolve

Reply:
This "pattern" has been applied by me or problame in the past. Since this is copying/mirroring `request` API, I think it's fine as is. Resolving.